Assets/Resources/Materials/Player Material.mat

```diff
@@ -73,5 +73,5 @@ Material:
     - _UVSec: 0
     - _ZWrite: 1
     m_Colors:
-    - _Color: {r: 1, g: 0, b: 0, a: 1}
+    - _Color: {r: 0, g: 1, b: 0, a: 1}
```

Comment:
what caused such a radical change? XD

Reply:
I have made the display color green/red to indicate if the mode is enabled or not. Some of the tools don't have a mode (like node, edge) so I made them green by default.

Reply:
ok, but this is player color as far as i can tell